Surface_mesh_segmentation/include/CGAL/internal/Surface_mesh_segmentation/Surface_mesh_segmentation.h

```diff
@@ -335,7 +335,7 @@ class Surface_mesh_segmentation
     // As far as I check: if, say, dihedral angle is 5, this returns 175,
     // if dihedral angle is -5, this returns -175.
     // Another words this function returns angle between planes.
-    double n_angle = to_double( ::CGAL::approximate_dihedral_angle(a, b, c, d) );
+    double n_angle = to_double( ::CGAL::approximate_dihedral_angle(a, b, c, d) ) * 180/CGAL_PI;
```

Comment:
the line below is doing / 180, simply remove it as well as your 180 here

Reply:
fixed.